tests/cloner_test.go

```diff
@@ -866,9 +867,12 @@ var _ = Describe("[rfe_id:1277][crit:high][vendor:cnv-qe@redhat.com][level:compo
 		Expect(err).ToNot(HaveOccurred())
 
 		By("Kill upload pod to force error")
-		_, errLog, err := f.ExecShellInPodWithFullOutput(targetNs.Name, utils.UploadPodName(targetPvc), "kill 1")
-		Expect(err).To(BeNil())
-		Expect(errLog).To(BeEmpty())
+		_, _, err = f.ExecShellInPod(utils.UploadPodName(targetPvc), targetNs.Name, "kill 1")
+		Expect(err).To(Or(
+			BeNil(),
+			WithTransform(errAsString, ContainSubstring("137"))))
+		// following log is needed just for an experiment, to be removed later.
+		fmt.Fprintf(GinkgoWriter, "INFO: err %v\n", err)
```

Comment:
removing this?

Reply:
this is actually part of experiment I want to see the problem happen, and then I'll be sure that it works, 

but after confirming I can remove it

Reply:
so far it is not failing often and mainly on crio, so It is hard to confirm, 
also it is not my priority right now so I am not trying too hard

Reply:
experiment finshed, this is now removed